package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "ethos-design-system",
-  "version": "1.1.4-n",
+  "version": "1.1.5-a",
   "description": "Ethos Design System v1",
-  "main": "./build/bundle.js",
+  "main": "./src/components/index.js",
```

Comment:
This has the effect that `import { Caption } from 'ethos-design-system'` should keep working with minimal changes elsewhere. When we want something else (e.g. a helper), I think the path still works as it does in your recent work in monorepo @roblevintennis.

Reply:
I see, and I'm guessing netlify uses `main` as it's entry point and that's why we'd see issues on netlify cms preview when we didn't rebundle and bump version (whereas locally CMS would pick up the changes).

Yeah, I like this whole direction 👍